package.json

```diff
@@ -63,6 +63,8 @@
     "redux-saga": "0.12.0",
     "source-map-support": "ncalexan/node-source-map-support#fileUrls-plus",
     "tmp": "0.0.29",
+    "transit-immutable-js": "0.6.0",
```

Comment:
I expect you want `transit` to rehydrate Immutable.js Records, but:

- is Transit's output itself stable across the life of the Tofino project/product?
- weigh heavily that Transit's output is not "hand hackable".  It's text, yes -- JSON, IIUC -- but it's not human readable or human editable.  At least, that's what I remember the last time I looked at some.

Is it worth being simpler here and just writing and reading JSON that _isn't_ Transit mangled?